### DISCUSSION TITLE
Tailwind CSS Syntax

### DISCUSSION THREAD

**Discussion by AbeEstrada** [helix-editor/helix#12200] [Q&A]
Is there a solution for this?
Does the LSP address this issue?

![tailwindcss](https://github.com/user-attachments/assets/47e0eac1-e13e-4993-98e6-65faa62e48b5)

**Comment by TornaxO7** [comment]
I think this issue is from your LSP, so what we need is the part of your language config which sets up your LSP for CSS.
Also take a look into the log what it's saying. You can execute add `-vvv` like `hx -vvv .` to get more into the logs.
